zerver/data_import/import_util.py

```diff
@@ -53,3 +59,56 @@ def build_recipient(type_id: int, recipient_id: int, type: int) -> ZerverFieldsT
         id=recipient_id,
         type=type)
     return recipient
+
+def process_avatars(avatar_list: List[ZerverFieldsT], avatar_dir: str, realm_id: int,
+                    threads: int, size: str='') -> List[ZerverFieldsT]:
```

Comment:
We should call `size` something more like `size_url_suffix`, since that's how it's used.